src/main/java/org/folio/rest/impl/AccountsAPI.java

```diff
@@ -2,18 +2,13 @@
 
 import static io.vertx.core.Future.succeededFuture;
 
-import io.vertx.core.AsyncResult;
```

Comment:
Is import ordering the only thing that has changed in this file?

Reply:
done.